### ISSUE TITLE
[docs] Recommended styling approach shown on all styling approaches

### ISSUE THREAD

**Issue by MonstraG** [mui/mui-x#13814]
### Search keywords

recommended styling approach

### Related page

https://mui.com/x/react-date-pickers/date-picker/

### Kind of issue

Broken demo

### Issue description

All three styling approaches seem to be "the recommended approach":

![image](https://github.com/user-attachments/assets/f8668fa8-6386-4910-8ae7-8b2bcf32b31b)

-----------------

![image](https://github.com/user-attachments/assets/2e740659-c078-44fc-97fc-3ed99e3ff9f9)

----------------

![image](https://github.com/user-attachments/assets/a26e2c5b-48f5-46ab-85b3-4ebf3d4e43ea)

(also true for desktop version)

**Comment by LukasTy** [comment]
Thank you for opening this issue.
Do you find the case of multiple "Recommended" approaches to styling rather than one (best) confusing?
In the case of `Static` Picker that is more or less the case.
But for the `Desktop` Picker it's pretty clearly stated that `theme` is recommended for styling. Is there something confusing with it? 🤔

**Comment by MonstraG** [comment]
> Do you find the case of multiple "Recommended" approaches to styling rather than one (best) confusing?
Yes, I read "Recommended" as "Best" here, because the entire page is recommendations on how to achieve certain things, that's what docs are.

I don't see any value having the same label on every approach - if everything is on discount, nothing is.

**Comment by LukasTy** [comment]
> I don't see any value having the same label on every approach - if everything is on discount, nothing is.

Thank you for the feedback. 🙏 
We'll discuss and see what improvement to clarify this part can be made. 😉
